packages/core/lib/src/widgets/html_table.dart

```diff
@@ -63,13 +63,13 @@ class HtmlTable extends MultiChildRenderObjectWidget {
 
 /// Companion data for table.
 class HtmlTableCompanion {
-  final _baselines = <int, List<_ValignBaselineRenderObject>>{};
+  final _baselines = <int?, List<_ValignBaselineRenderObject>>{};
```

Comment:
```suggestion
  final _baselines = <int, List<_ValignBaselineRenderObject>>{};
```

(`_ValignBaselineRenderObject` needs adjustments)